code/modules/ruins/spaceruin_code/clownstation.dm

```diff
@@ -0,0 +1,56 @@
+
+/obj/item/paper/fluff/ruins/clownstation/genesisnotice
+	name = "visitor notice"
+	info = "The Honkspel of Honkesis is often disputed, and at this monastery we don't recognize it. It's become clear in recent Honkspel scholarship that the book was added at a later date merely for descrimination agaisnt mimes, who we feel, despite our differences, are still good people. The book of Honkesis also presents a truly unrealistic depiction of the creation of the universe. We are religious, but we do not deny science. This book was truly a piece of political writing attempting to pass as the word of the Honkmother."
+
+/obj/item/paper/fluff/ruins/clownstation/murderersprayer
+	info = "Honkmother, I'm so dearly sorry for the murder I have committed.  I didn't mean to spill innocent blood today. I didn't want to see him die. I had to do it. He caught me red handed. Dammit, his face haunts me. Forgive me honkmother. Please forgive me. I know I have sinned. I didn't want to be a criminal. I went to clown college to cheer people up, but money is tight, and I can't keep sharon and the kids alive on bananas. Maybe if I had a stronger father figure I wouldn't have turned to crime, but my pappy died in the war. I don't know, I can't blame him. I'm weak. I've always been weak. I spent my teenage years out of my mind on rainbow weed and gargle blasters. Maybe I wasn't meant to face the world. Please have mercy, I beg you."
+
+/obj/item/paper/fluff/ruins/clownstation/letterhome
+	info = "My dear family, I have quite the tale to write home to you about. It seems a small group of mime insurgents stowed away somewhere of our ship. It seemed they might get the upper hand, as they were striking in the night and we didn't know where they were coming from. The power of prayer works wonders, my dear family. We prayed for help and the honkmother listened. A divine figure arrived with a flash of yellow light and a bone shaking honk. He smote the insurgents with holy force. I do hope he sticks around. He's always welcome in our monastery."
+
+/turf/closed/wall/fakebananium
+	name = "imitation bananium wall"

```

Comment:
could it be possible to allow clowns to crate imitation bananium walls by using banana peels on girders or something?